src/main.c

```diff
@@ -593,6 +592,19 @@ void ui_color_init() {
     }
 }
 
+static void center_fkey_header() {
+    int width = sizeof(fkey_header) - 1;
+    if (strlen(fkey_header) == width) {
+	return;     // already OK
+    }
+    int left_padding = (width - strlen(fkey_header)) / 2;
+    int right_padding = width - strlen(fkey_header) - left_padding;
+    char tmp[sizeof(fkey_header)];
+    strcpy(tmp, fkey_header);
+    sprintf(fkey_header, "%s%s%s",
+	    spaces(left_padding), tmp, spaces(right_padding));
+}
+
```

Comment:
I had done it in `show_header_line`, but that is another way. Fine.

Reply:
This way it's done once at start. All else stays as-is.

Reply:
- Imho, centering would possibly be better off in parse_logcfg, needed only once per config-file re-read
-  just because the next idea is almost completely implemented by now, needing only to conditionally zero out left- or right padding, would it be too much Left|Centered|Right aligning instead of centering, FKEY_HEADER=[L:|C:|R:]text, defaulting to "C:" ?

Reply:
Centering will be just a nice to have once we implement leading/trailing whitespace for config values. This is on my list. With whitespace the user will have better control on the appearance of the text.

Reply:
Thank you very much, fully agree on whitespaces with you.